media/src/main/java/edu/artic/media/audio/AudioPlayerService.kt

```diff
@@ -502,6 +502,26 @@ class AudioPlayerService : DaggerService(), PlayerService {
         analyticsTracker.reportEvent(EventCategoryName.PlayBack, AnalyticsAction.playbackInterrupted, (currentTrack as BehaviorSubject).value?.title.orEmpty())
         audioControl.onNext(AudioPlayerService.PlayBackAction.Stop())
     }
+
+    /**
+     * Returns currently playing [ArticAudioFile].
```

Comment:
Hmm. May be worth mentioning in the comment that tours don't need/have these.

Reply:
Added the docs mentioning that method is intended to be used for `ArticObject` only.